server/src/routes/feeds.rs

```diff
@@ -247,6 +252,7 @@ fn create_reply_and_mention_items(
         r.id
       );
       build_item(&r.creator_name, &r.published, &reply_url, &r.content)
+        .unwrap_or_else(|_| panic!(location_info!()))
```

Comment:
I couldn't figure out a good way to do these otherwise.

Reply:
You should remove the `unwrap_or_else()`, and replace the `.collect()` below with `.collect::<Result<Vec<Item>, LemmyError>>()?`.

Same thing below in line 269.

Reply:
Ah thanks, that's a much easier way to do it. I tried and failed at making the top level vectors results.